pkg/tanka/parallel.go

```diff
@@ -0,0 +1,82 @@
+package tanka
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/grafana/tanka/pkg/spec/v1alpha1"
+)
+
+const parallel = 8
+
+// parallelLoadEnvironments evaluates multiple environments in parallel
+func parallelLoadEnvironments(paths []string, opts ParallelOpts) (envs []*v1alpha1.Environment, err error) {
```

Comment:
With `parallelLoadEnvironments` now being private, `ParallelOpts` should be as well

Reply:
It is part used in `ExportOpts` and then used again in `cmd/tk/export.go`. I prefer to keep it as part of the Export API.